app/bundles/CampaignBundle/Entity/Summary.php

```diff
@@ -18,32 +18,32 @@ class Summary
     private $id;
 
     /**
-     * @var \DateTimeInterface|null
+     * @var \DateTimeImmutable|null
      **/
     private $dateTriggered;
 
     /**
-     * @var int|null
+     * @var int
      */
     private $scheduledCount = 0;
 
     /**
-     * @var int|null
+     * @var int
      */
     private $triggeredCount = 0;
 
     /**
-     * @var int|null
+     * @var int
      */
     private $nonActionPathTakenCount = 0;
 
     /**
-     * @var int|null
+     * @var int
      */
     private $failedCount = 0;
 
     /**
-     * @var Event|null
+     * @var Event
```

Comment:
It used in `ManyToOne` in [this line](https://github.com/mautic/mautic/blob/5.x/app/bundles/CampaignBundle/Entity/Summary.php#L75), It can not be null

Reply:
I relaxed `phpstan` checks to allow `null`, this is one is fixed in commit 03915a348dadafe9699c8f696bcd18f6e3ed3b5d